src/proofPoints.ts

```diff
@@ -677,6 +677,16 @@ class ProofPointRegistry {
     );
   }
 
+  private didToUrl(did: string): string {
```

Comment:
Should this be `DID`, as its an acronym? Especially important given that "did" is also a word.

Could add some method docs explaining the DID formats this is parsing with examples.

Reply:
Docs added. 
I prefer to treat acronyms as words in the context of camel case, otherwise you lose the readability of camel case. Also that is the standard so far in this library and might be a big change to switch.